### DISCUSSION TITLE
home_assistant with MQTT

### DISCUSSION THREAD

**Discussion by swamp2k** [roflcoopter/viseron#704] [Q&A] [answered]
Fiddling around with the MQTT home_assistant option but I seem to be doing something wrong in the formatting.

I can't find it explicitly in the docs, but I took a guess that it was:
home_assistant: true 
Seems I was wrong. that gives me an error
`[2024-01-20 21:42:26] [ERROR   ] [viseron.components] - Error validating config for component mqtt: expected dict or None for dictionary value @ data['mqtt']['home_assistant']. Got True`

My MQTT section is fairly simple:
mqtt:
  broker: 192.168.50.90
  port: 1883
  home_assistant: true

Hoping anyone can clear this up for me :)

**Comment by roflcoopter** [comment, marked as answer]
It should just be like this:

```yaml 
mqtt:
  broker: 192.168.50.90
  port: 1883
  home_assistant:
